Gulpfile.js

```diff
@@ -33,7 +43,7 @@ gulp.task('sass:lint', function() {
 });
 
 gulp.task('sass:build', function() {
-  gulp.src('./src/sass/**/style.scss')
+  return gulp.src('./src/sass/**/style.scss')
```

Comment:
This lets Gulp know when the task is actually done. That way, runSequence can run tasks in a specified order, or gulp.task can run the callback function when the dependent tasks are done. Otherwise, it might happen concurrently.

I was having a bug where my deploy script would upload code to Firebase before the build was actually done. This was the fix.